lib/utils/makeValidator.js

```diff
@@ -129,8 +129,8 @@ const makeValidator = (mainSchema, subSchemas) => {
   });
   return {
     validate: definition => {
-      const { errors, ...results } = v.validate(definition, mainSchema);
-      const allErrors = errors.concat(
+      const results = v.validate(definition, mainSchema);
+      const allErrors = results.errors.concat(
```

Comment:
Nice. Not a blocker by any means, but should have have something to indicate that `results` is a class and not a plain object? I suppose we can just peek at the API for `jsonschema`.

Reply:
yeah, could change the variable name or something if we wanted. I don't think there should be an expectation that someone editing this code knows the implementation details of jsonschema.

I think it's probably fair game to return an object that has the `valid` boolean, but isn't the actual class. We've codified the behavior that we rely on (`valid is always defined`) so past that we're probably good pas that.